ssg/constants.py

```diff
@@ -4,11 +4,35 @@
 import datetime
 import os.path
 
-product_directories = ['debian8', 'debian9', 'fedora', 'ol7', 'ol8', 'opensuse', 'rhel6',
-                       'rhel7', 'rhel8', 'sle11', 'sle12', 'ubuntu1404',
-                       'ubuntu1604', 'ubuntu1804', 'wrlinux8', 'wrlinux1019', 'rhosp10',
-                       'rhosp13', 'chromium', 'eap6', 'firefox', 'fuse6', 'jre', 'ocp3',
-                       'example']
+product_directories = [
+    'chromium',
+    'debian8',
+    'debian9',
+    'eap6',
+    'example',
+    'fedora',
+    'firefox',
+    'fuse6',
+    'jre',
+    'ocp3',
+    'ocp4',
+    'ol7',
+    'ol8',
+    'opensuse',
+    'rhel6',
+    'rhel7',
+    'rhel8',
```

Comment:
I imagined these to be on the same line. It can apply to other versioned products too.
e.g. 
```
    'debian8', 'debian9'`,
    `ocp3, `ocp4`,
````

Reply:
force pushed last commit, please check